src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="search"
+    method="get"
+    data-qa="big"
+    class="form bigForm"
+    >
+      <label class="bigLabel">
+        <input
+        type="text"
+        class="field bigSearch"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        >
+      </label>
```

Comment:
```suggestion
      <label class="bigLabel">
        <input
          type="text"
          class="field"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
      </label>
```

Reply:
same here

Reply:
I fixed indentation, but I need class bigSearch, as well as smallSearch. They have different property values and I just can't join them. All the same values I wrote in class="field"